src/Translatable/Validation/Rules/TranslatableUnique.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Astrotomic\Translatable\Validation\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+
+/**
+ * Custom unique validation for translatable attributes
+ * 
+ * @author Amjad BaniMattar <amjad.banimattar@gmail.com>
+ */
+class TranslatableUnique implements ValidationRule
```

Comment:
Pretty much the same as for the exists rule.

Reply:
Yes, but here the logic is reversed. Throw validation Exception if the value exists whereas `TranslatableExists` throws validation exception if the value dose not exist.

Update code to inherit the TranslatableExists to eliminate the repeated code